testinfra/modules/salt.py

```diff
@@ -45,6 +45,8 @@ def __call__(self, function, args=None, local=False):
             cmd = "salt-call --out=json"
             if local:
                 cmd += " --local"
+            if config is not None:
+                cmd += " -c {0}".format(config)
```

Comment:
Hello, thanks for contributing !

Looks good to me except that if "config" need to be shell escaped (if it contains spaces or something else)

I'll do:

```python
cmd_args = [function]
[...]
if config is not None:
    cmd += " -c %s"
    cmd_args.append(config)
cmd += " %s" + len(args) * " %s"
cmd_args += args
```

Let me know if you have time to fix this, if not I'll be happy to modify your commit and to merge it. Thanks !

Reply:
if we put the function first in cmd_args, it will be passed to `-c` instead, but I can do the string escape stuff.